gips/datahandler/logger.py

```diff
@@ -33,8 +33,10 @@ def __new__ (cls):
     def __init__(self):
         self.extra = {}
         try:
+            # TODO this can't work for RQ
             self.extra['jobid'] = os.environ['PBS_JOBID']
         except:
+            # TODO is this correct?
```

Comment:
Sorry that's a stale TODO, I later tested it and discovered it worked.  It's easy to take this out.